code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -26,22 +26,25 @@
 /obj/item/implant
 	name = "implant"
 	icon = 'icons/obj/implants.dmi'
-	icon_state = "generic" //Shows up as the action button icon
+	icon_state = "generic" //Shows up as a auto surgeon, used as a placeholder when a implant doesn't have a sprite
 	origin_tech = "materials=2;biotech=3;programming=2"
-
 	actions_types = list(/datum/action/item_action/hands_free/activate)
+	item_color = "b"
```

Comment:
🅱️ ?

Reply:
old code moment, I'll see what the consequences of removing this is, although there should be none.

Reply:
changed to "black"